test/configCases/dll-plugin/5-use-dll-with-package-name/index.js

```diff
@@ -0,0 +1,43 @@
+const should = require("should");
```

Comment:
Filed test seems to be complaining that you're using const here. if you add ```"use strict";``` at the top of this file it'll probably resolve the issue.